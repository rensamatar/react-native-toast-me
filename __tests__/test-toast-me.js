import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'
import ToastMe from '../src/index'

it('renders correctly', () => {
  const tree = renderer.create(
    <ToastMe message={'Hello world!'} type={'success'} isShow={true} />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})