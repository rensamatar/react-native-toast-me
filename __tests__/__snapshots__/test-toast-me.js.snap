// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  collapsable={undefined}
  style={
    Object {
      "backgroundColor": "#4CAF50",
      "height": 0,
      "justifyContent": "center",
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
      "transform": Array [
        Object {
          "translateY": -70,
        },
      ],
      "zIndex": 1,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    numberOfLines={2}
    style={
      Object {
        "color": "white",
        "fontSize": 16,
        "fontWeight": "bold",
        "marginHorizontal": 10,
        "textAlign": "center",
      }
    }
  >
    Unknown
  </Text>
</View>
`;
